docs/docs/lavague-qa/usage.md

```diff
@@ -0,0 +1,131 @@
+# LaVague QA: Usage guide
+
+!!! warning "Early release"
+    LaVague QA is still a work in progress and may contain bugs. Join our [community of amazing contributors](https://discord.gg/invite/SDxn9KpqX9) to help make this tool more reliable!
+
+## Installation
+
+LaVague QA uses `gpt-4o` models by default, as such, you will need to define `OPENAI_API_KEY` as an environment variable before running LaVague. 
+
+### Install with pip
+
+Install the latest release of our package with `pip`
+
+```
+pip install lavague-qa
+```
+
+### Install from source
+
+Install from the latest repository source
+
+Clone the [LaVague](https://github.com/lavague-ai/LaVague) repository
+
+```sh
+git clone https://github.com/lavague-ai/LaVague.git
+```
+
+Navigate to the `lavague-qa` package
+```
+cd LaVague/lavague-qa
+```
+
+Install with pip
+```
+pip install -e .
+```
+
+
+## Default usage
+
+Run `lavague-qa --help` to see all available arguments
+
+```
+Usage: lavague-qa [OPTIONS]
+
+Options:
+  -u, --url TEXT      URL of the site to test
+  -f, --feature TEXT  Path to the .feature file containing Gherkin
+  -llm, --full-llm    Enable full LLM pytest generation
+  -c, --context TEXT  Path of python file containing an initialized context
+                      and token_counter. Defaults to OpenAI GPT4o
+  -h, --headless      Enable headless mode for the browser
```

Comment:
I think it's worth adding a line below explaining what this means as it's a very important option + add a quick note in the QT that you can run with headless or non-headless mode